code-challenges.test.js

```diff
@@ -34,21 +56,93 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
+// Psuedo code:
+// Start on the function by naming a function divideByThree
+// Write out all the syntax for the function
+// Use two if else statments
+// use the % modulator to check if a number is divisible by 3
+// return is divisible by 3 if true
+// return is not divisible by 3 if not true
+
+// const divideBYThree = () => {
+//     if( object.number % === 0){
+//         return `${object.number} is divisible by three.`
+//     } else {
+//         return`${object.number} is not divisibly by three`
+//     }
+// }
+
+// console.log(divideByThree(object1))
+
+// I couldnt get the test to pass. It says unexpected number when I console.logged it. I think something is wrong with my test i dont think its in my actual function. 
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// Psuedo code:
+// Start with describe a function
+// "It" will return all the words in the array capitalized
+// Make two expect statements one for the first const one for the second
+// run test using $yarn jest watch it fail
+
+// describe("capitilizer", () => {
+//     it(returns every string in the array in capital letters, () => {
+//         const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+//         const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+//         expect(capitilizer(randomNouns1)).toEqual(["STREETLAMP", "POTATO", "TEETH", "CONCLUSION", "NEPHEW"])
```

Comment:
use those expected outputs on lines 102 and 104 to help you with your expect states